MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -318,7 +319,8 @@ - (void)detachAVPlayer {
 }
 
 - (void)monitorAVPlayerItem {
-    if (!_automaticVideoChange) {
+    if (!_automaticVideoChange && !_didTriggerManualVideoChange) {
+        _didTriggerManualVideoChange = false;
```

Comment:
I'm not sure what the purpose of this change is other than to return since `_didTriggerManualVideoChange` must already be `false`.
```suggestion
```

Reply:
Great catch Scott, actually. That should be without the `!`, but I'm now confused as to the tests passing...

Reply:
I think the `!` is correct we want to return if either:
1. _automaticVideoChange is disabled
2. _didTriggerManualVideoChange was false

The reset should be here and we need a test case to catch this
```
if (!_automaticVideoChange && !_didTriggerManualVideoChange) {
        return;
    }
    if (_playerItem) {
        if (_didTriggerManualVideoChange) {
            _didTriggerManualVideoChange = false;
        }
        [self stopMonitoringAVPlayerItem];
        [self.playDispatchDelegate videoChangedForPlayer:_name];
        ```